.ci/oci-devworkspace-happy-path.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+#
+# Copyright (c) 2012-2021 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+
+# exit immediately when a command fails
+set -e
+# only exit with zero if all commands of the pipeline exit successfully
+set -o pipefail
+# error on unset variables
+set -u
+
+export OPERATOR_REPO=$(dirname $(dirname $(readlink -f "$0")));
+export HAPPY_PATH_POD_NAME=happy-path-che
+export HAPPY_PATH_DEVFILE='https://github.com/l0rd/spring-petclinic/tree/devfile2'
+source "${OPERATOR_REPO}"/.github/bin/common.sh
+source "${OPERATOR_REPO}"/.github/bin/oauth-provision.sh
+
+# Stop execution on any error
+trap "catchFinish" EXIT SIGINT
+
+deployChe() {
+  cat > /tmp/che-cr-patch.yaml <<EOL
+spec:
+  devWorkspace:
+    enable: true
+  server:
+    customCheProperties:
+      CHE_FACTORY_DEFAULT__PLUGINS: ""
+      CHE_WORKSPACE_DEVFILE_DEFAULT__EDITOR_PLUGINS: ""
+EOL
+
+  cat /tmp/che-cr-patch.yaml
+
+  chectl server:deploy --che-operator-cr-patch-yaml=/tmp/che-cr-patch.yaml -p openshift --batch --telemetry=off --installer=operator
+}
+
+runTest() {
+  deployChe
+
+  # patch happy-path-che.yaml 
+  ECLIPSE_CHE_URL=http://$(oc get route -n "${NAMESPACE}" che -o jsonpath='{.status.ingress[0].host}')
+  TS_SELENIUM_DEVWORKSPACE_URL="${ECLIPSE_CHE_URL}/#${HAPPY_PATH_DEVFILE}"
+  sed -i "s@CHE_URL@${ECLIPSE_CHE_URL}@g" ./.ci/openshift-ci/happy-path-che.yaml
+  sed -i "s@WORKSPACE_ROUTE@${TS_SELENIUM_DEVWORKSPACE_URL}@g" ./.ci/openshift-ci/happy-path-che.yaml
+  cat ./.ci/openshift-ci/happy-path-che.yaml
```

Comment:
I am wondering if it is better to use 
`${OPERATOR_REPO}/.ci` instead of `./.ci`